client/src/components/shared/Layout/LayoutTopBarCalendar/CalendarItem/CalendarItem.tsx

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Instead of disabling them for the entire file, please disable them specifically in some place(s). Here, if we ever introduce another rule-breaking change we won't know.